packages/Chatdown/bin/chatdown.js

```diff
@@ -131,6 +131,9 @@ async function runProgram() {
         process.stderr.write(chalk.default.white(`to update.\n`));
     }
 
+    process.stderr.write(chalk.default.yellow(`\nWARNING:  chatdown is deprectated.  For more information refer to https://aka.ms/botframework-cli.\n`));
```

Comment:
Why is it deprecated? There is no information on:
https://aka.ms/botframework-cli

Reply:
Thanks for your question @rufreakde. As an effort to make our tools easier to use and keep everything organized we are currently working on the new set of tools which is botframework-cli, following the azure cli pattern. Chatdown is already available and ready to use, you should be able to install it by running npm i -g @microsoft/botframework-cli